sphinx/source/members/adding_member.rst

```diff
@@ -0,0 +1,55 @@
+
+.. code-block:: bash
+
+    $ keygenerator.sh --name=member_name --gen-key-share
+    -- Generating identity private key and certificate for participant "member_name"...
+    Identity curve: secp384r1
+    Identity certificate generated at:   member_name_cert.pem (to be registered in CCF)
+    Identity private key generated at:   member_name_privk.pem
+    -- Generating key share pair for participant "member_name"...
+    Key share public key generated at:   member_name_kshare_pub.pem (to be registered in CCF)
+    Key share private key generated at:  member_name_kshare_priv.pem
+
+The member’s private keys (e.g. ``member_name_privk.pem`` and ``member_name_kshare_priv.pem``) should be stored on a trusted device while the certificate (e.g. ``member_name_cert.pem``) and public key share (e.g. ``member_name_kshare_pub.pem``) should be registered in CCF by members.
+
+.. note:: See :ref:`developers/cryptography:Algorithms and Curves` for the list of supported cryptographic curves for member identity.
+
+Registering a New Member
+------------------------
+
+Once the new member's keys and certificate have been generated, the existing consortium should register the new member public information in CCF, following the usual propose and vote procedure.
+
+The :ref:`members/proposals:Submitting a New Proposal` section describes the steps that members should follow to register a new member.
+
+Activating a New Member
+-----------------------
+
+A new member who gets registered in CCF is not yet able to participate in governance operations. To do so, the new member should first acknowledge that they are satisfied with the state of the service (for example, after auditing the current constitution and the nodes currently trusted).
+
+First, the new member should update and retrieve the latest state digest via the ``members/updateAckStateDigest`` command. In doing so, the new member confirms that they are satisfied with the state of the service.
+
+.. code-block:: bash
+
+    $ curl https://<ccf-node-address>/members/updateAckStateDigest  --cacert networkcert.pem --key new_member_privk.pem --cert new_member_cert.pem --header "Content-Type: application/json" --data '{"jsonrpc":"2.0", "id":0, "method":"members/updateAckStateDigest", "params":{}}'
+    {"commit":57,"global_commit":56,"id":0,"jsonrpc":"2.0","result":[<state_digest>],"term":2}
+
+
+Then, the new member should sign the state digest returned by the ``members/updateAckStateDigest`` command with their private key and issue the signature via the ``members/ack`` command.
+
+.. code-block:: bash
+
+    $ curl https://<ccf-node-address>/members/ack  --cacert networkcert.pem --key new_member_privk.pem --cert new_member_cert.pem --header "Content-Type: application/json" --data '{"jsonrpc":"2.0", "id":0, "method":"members/updateAck", "params":{"sig":"[<signature_over_state_digest>]"}}'
```

Comment:
I'm trying to find a way to sign the root via the openssl CLI. It shouldn't block this PR to be merged though.

Reply:
Shouldn't this be a scurl.sh?

Reply:
Now addressed.